deploy/Makefile

```
@@ -69,8 +69,11 @@ clean-spoke:
 ssh:
 	@./aws-hypervisor/scripts/ssh.sh
 
-info:
-	@./aws-hypervisor/scripts/print_instance_data.sh
+status:
+	@./openshift-clusters/scripts/status.sh
+
+# Alias kept for muscle memory: 'make info' output is part of 'make status'
+info: status
```

checkmake (0.3.2)

[warning] Target "info" should be declared PHONY.

(phonydeclared)